src/NBS.Appointments.Service/Models/AvailabilityResponse.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Newtonsoft.Json;
+
+namespace NBS.Appointments.Service.Models
+{
+    public class AvailabilityResponse
```

Comment:
Any reason to move these from being a nested class? It is only ever used as part of the AvailabilityQueryResponse so seems to make sense